libnetdata/sysinc.h

```diff
@@ -0,0 +1,152 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
```

Comment:
Please, add to this file these code:

```
#ifdef __cplusplus
extern "C" {
# endif
```
and

```
# ifdef __cplusplus
}
# endif
```

I am asking this, because we had an user contribution last year that added them, because our headers were not 100% compatible with c++ compilers. Considering that good part, if not all the code, is coming from `libnetdata.h` and it was present inside these conditions, it is possible that c++ developers could have the problem again.

I already tested this PR on `Slackware current` and `FreeBSD 12.2` and I did not observe any warning, after this change I will approve it.